examples/systemcalls/systemcalls.c

```diff
@@ -71,6 +215,8 @@ bool do_exec(int count, ...)
 */
 bool do_exec_redirect(const char *outputfile, int count, ...)
 {
+    openlog("system_calls", LOG_PID, LOG_USER);
```

Comment:
@traptibalgi think about how you could avoid duplicating logic with the function above.